orbit/pkg/token/readwriter.go

```diff
@@ -44,15 +48,20 @@ func (rw *ReadWriter) Rotate() error {
 		return fmt.Errorf("generate identifier: %w", err)
 	}
 
-	err = os.WriteFile(rw.Path, []byte(id.String()), constant.DefaultSystemdUnitMode)
+	err = os.WriteFile(rw.Path, []byte(id.String()), constant.DefaultWorldReadableFileMode)
 	if err != nil {
 		return fmt.Errorf("write identifier file %q: %w", rw.Path, err)
 	}
 
+	// ensure the file is readable by other processes, os.WriteFile does not
+	// modify permissions if the file already exists
```

Comment:
Ah good catch!, I missed this case.

Reply:
yes, I missed it too despite staring at this code for long.  Thanks for the review.